config/config.go

```diff
@@ -44,6 +44,21 @@ var ErrNoAnalyzerSpecified = errors.New("No analyzer specified in the configurat
 var (
 	cfg           *viper.Viper
 	relocationMap = map[string][]string{
+		"analyzer.storage.flow.backend": {
```

Comment:
let's focus this pr only on connection per index
PR #968 will address new config namespace

Reply:
ok, i have take code related to relocation, as it is more complete that #968 - I have created a new PR from branch `storage-relocate-backend` - @safchain - please suggest how/who will handle.